src/app/client-view/training-view/training-day/exercise/exercise.component.ts

```diff
@@ -40,6 +41,7 @@ export class ExerciseComponent implements OnInit {
           Validators.min(0),
           Validators.max(10),
         ]),
+        notes: new FormControl('')
```

Comment:
To leave the form blank by default it should be:
```suggestion
        notes: new FormControl(null)
```